tests/guest_tools/win/__init__.py

```diff
@@ -147,6 +147,19 @@ def vif_set_dns(vif: VIF, nameservers: List[str]):
     )
 
 
+def wait_for_vm_xenvif_offboard(vm: VM):
+    # Xenvif offboard will reset the NIC, so need to wait for it to disappear first
+    wait_for(
+        lambda: strtobool(
+            vm.execute_powershell_script(
+                r'$null -eq (Get-ScheduledTask "Copy-XenVifSettings" -ErrorAction SilentlyContinue)', simple_output=True
+            )
+        ),
+        timeout_secs=300,
+        retry_delay_secs=30,
+    )
+
+
```

Comment:
Wouldn't those libraries (both `guest_tools/win/__init__.py` and `guest_tools.py` be better moved into `lib/`?

Reply:
All moved to `lib/windows/`.